### PR TITLE
Automate sbt sonatypeReleaseAll [skip ci]

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

**Comment by olafurpg** [comment]
I'm not 100% sure if this is correct, but it's worth a shot.

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,11 @@ script:
 jdk:
   - oraclejdk8
 
+stages:
+  - name: test
+  - name: release
+    if: (branch = master) AND (type = push)
```
Comment (xeno-by) [inline .travis.yml]: This seems to be inconsistent with `if [[ "$TRAVIS_TAG" ]];` below?

Comment (olafurpg) [reply .travis.yml]: Good catch! This is copied from the metals repo. I removed the `branch = master` guard but kept `type = push`

File: .travis.yml [hunk 2]
```diff
@@ -45,6 +50,8 @@ matrix:
       - CI_SCALA_JS: true
     - env:
       - CI_TEST: mima
+    - stage: release
+      script: if [[ "$TRAVIS_TAG" ]]; then sbt "sonatypeReleaseAll"; else echo "skipping release"; fi
```
Comment (xeno-by) [inline .travis.yml]: How about `"only releasing on tag pushes"`?

Comment (olafurpg) [reply .travis.yml]: Done.

**Pull request merged**